classes/class-pmpro-wisdom-integration.php

```diff
@@ -215,7 +215,7 @@ public function bypass_local_tracking( $is_local = false ) {
 	 * @return string
 	 */
 	public function override_notice() {
-		return __( 'Share your usage data to help us improve Paid Memberships Pro. We use this data to analyze how our plugin is meeting your needs and identify new opportunities to help you create a thriving membership business. You can always visit the Advanced Settings and change this preference. <a href="https://www.paidmembershipspro.com/privacy-policy/usage-tracking/">Read more about what data we collect</a>.' );
+		return __( 'Share your usage data to help us improve Paid Memberships Pro. We use this data to analyze how our plugin is meeting your needs and identify new opportunities to help you create a thriving membership business. You can always visit the Advanced Settings and change this preference. <a href="https://www.paidmembershipspro.com/privacy-policy/usage-tracking/">Read more about what data we collect</a>.', 'paid-memberships-pro' );
```

Comment:
For strings with links, we don't want the URL to be translatable. People can put in their own links and it's easy to miss.

We'd need only the text between the `<a href` translatable.

Reply:
We can just use the sprintf and a placeholder then.